foundry.toml

```diff
@@ -0,0 +1,3 @@
+[profile.default]
+src = "contracts"
+test = "test-foundry"
```

Comment:
Please insert a newline at the end of the file. There is a setting in VSCode to not have to worry about this: it is called "Files: Insert Final Newline"